Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM alpine:3.11 as base
+
+RUN apk add curl
+
+RUN apk add python3
```

Comment:
You could just use a base image that has python installed.

Currently you're not controlling what vesion of Python you're using.

Reply:
will do, ty

Reply:
You still haven't done this. I meant use e.g. `FROM python:3.9.4-slim-buster as base` for your base and then remove this line. 

Or `FROM python:3.9-alpine as base` if you want to stick to Alpine.

Reply:
Thank you, I have tried some other image I thought worked but it didn't so I left it. should have mentioned I couldn't do it, thank you, sorry